### PR TITLE
Add bob.db.base recipe

### PR DESCRIPTION
**Pull request by 183amir**

### CODE REVIEW

File: recipes/bob.db.base/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  - nose
+
+about:
+  home: http://github.com/bioidiap/bob.db.base
+  license: Modified BSD License (3-clause)
+  summary: Bob's Basic Database API
+
+extra:
+  recipe-maintainers:
+  - 183amir
```
Comment (pelson) [inline recipes/bob.db.base/meta.yaml]: your indentation in this recipe is off by a little bit (in a number of places).

Comment (jakirkham) [inline recipes/bob.db.base/meta.yaml]: Yeah, this goes beyond style here. It could actually cause problems for various parsers in the automation process.

Comment (183amir) [inline recipes/bob.db.base/meta.yaml]: It was like this for all `bob` packages. :)

**Comment by 183amir** [comment]
@pelson could you please merge this too? And: https://github.com/conda-forge/staged-recipes/pulls/183amir

**Comment by pelson** [comment]
> Yeah, this goes beyond style here. It could actually cause problems for various parsers in the automation process.

I'm curious what the implications of this are so was waiting for some more discussion. @183amir - are you happy to re-apply your script to the feedstocks to bring the recipes into a nicer form-factor (even if that is a non-functional change?).

**Comment by 183amir** [comment]
@pelson I have fixed this for future recipes. And I'll be happy to do this for feedstocks as well. Please go ahead and merge these two pull requests and I will fix them on feedstocks with the rest of bob packages.

**Comment by pelson** [comment]
Thanks @183amir!

**Pull request merged**